site/en/docs/devtools/rendering/index.md

```diff
@@ -0,0 +1,356 @@
+
+## Highlight ad frames
+
+To check if frames have been tagged as ads:
+
+1. [Open the **Rendering** tab](#open-rendering)  an the [demo page](https://heavy-ads.glitch.me/) and check **Highlight ad frames**.
+1. Observe the ad frame highlighted in red.
+
+{% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/UuQMD7ifcGe1n6nFdYth.png", alt="Ad highlighted in red", width="800", height="490" %}
+
+## View Core Web Vitals
+
+[Web Vitals](https://web.dev/vitals/) is an initiative by Google to provide unified guidance for quality signals that are essential to delivering a great user experience on the web.
+
+Core Web Vitals are the subset of Web Vitals that apply to all web pages. Each of the Core Web Vitals represents a distinct facet of the user experience, is measurable in the field, and reflects the real-world experience of a critical user-centric outcome. The Core Web Vitals are:
+
+- **[Largest Contentful Paint (LCP)](https://web.dev/lcp/)**: measures _loading_ performance.
+  To provide a good user experience, LCP should occur within **2.5 seconds** of
+  when the page first starts loading.
+- **[First Input Delay (FID)](https://web.dev/fid/)**: measures _interactivity_. To provide a
+  good user experience, pages should have a FID of **100 milliseconds** or less.
+- **[Cumulative Layout Shift (CLS)](https://web.dev/cls/)**: measures _visual stability_. To
+  provide a good user experience, pages should maintain a CLS of **0.1.** or
+  less.
+
+To view the Core Web Vitals as an overlay in the top right corner of the viewport:
+
+1. [Open the **Rendering** tab](#open-rendering) and check **Core Web Vitals**.
+1. Scroll the page to reveal all layout shifts and perform an interaction, for example, click a button, open a tab, or enter text in a textbox.
+
+   {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/HW8iWmtzvC3S0uuQdcsp.png", alt="ALT_TEXT_HERE", width="800", height="311" %}
+
+Green Core Web Vitals indicate that your page is in good shape. Yellow or red vitals need attention.
+
+## Disable local fonts
+
+Check if the local font alternatives work as expected by disabling `local()` sources in `@font-face` rules.
+
+<style>
+    @font-face {
+      font-family: "MyFont";
+      src: local("Courier New"), url(https://fonts.gstatic.com/s/roboto/v29/KFOmCnqEu92Fr1Mu4mxKKTU1Kg.woff2) format('woff2');
+    }
+    .text-box {
+      font-family: 'MyFont';
+    }
+</style>
+<div class="text-box">Chrome displays this sentence in Courier New if it finds this font on your computer.</div>
+
+To disable local fonts:
+
+1. [Open the **Rendering** tab](#open-rendering) and check **Disable local fonts**.
+1. <span class="material-icons">refresh</span> Reload the page and observe the font of the sentence above change to Roboto.
+
+## Emulate a focused page
+
+If you switch focus from the current page to DevTools, some overlay elements automatically hide if they are triggered by focus. For example, dropdown lists, menus, or date pickers.
+
+To emulate a focused page:
```

Comment:
Can improve on the example as the effect is not obvious.A dropdown example is better or a demo video. 

Refer to demo at 2:00. https://portal.gitnation.org/contents/debugging-with-chrome-devtools